keymanapp/keyman
# chore(common): Add Crowdin strings for Mon

This adds crowdin strings for Mon (`mnw-MM` written with Burmese script).
Not available for iOS or macOS.

TODO for @sgschantz 
- [x] See if iOS can add mnw-MM locale
- [x] See if macOS can add mnw-MM locale

## User Testing
Test the Keyman apps can change to UI in Mon

* **TEST_ANDROID** 
1. On an Android device/emulator, load the PR build of Keyman for Android
2. In the Keyman app, go to Settings --> Display language --> မန် (Mon)
3. Exit the menu and wait for the app to refresh
4. Verify the UI is in Mon

![android mon](https://github.com/keymanapp/keyman/assets/7358010/e563e7ff-3540-4636-97b4-a82624c56d4a)


* **TEST_WINDOWS**
1. Install the PR build of Keyman for Windows
2. In Keyman configuration, change the UI to "မန် (Mon)"
3. Verify the UI is in Mon

* **TEST_LINUX**
1.Start Keyman configuration with: LANGUAGE=mnw_MM km-config
2. Verify Keyman UI is in Mon
